src/status_im/profile/core.cljs

```diff
@@ -101,8 +101,12 @@
       {:db            (-> db
                           (assoc :contacts/identity identity)
                           (assoc :contacts/ens-name ens-name))
-       :json-rpc/call [{:method      "wakuext_buildContact"
-                        :params      [identity]
+       :json-rpc/call [{:method      (if ens-name
+                                       "wakuext_buildContactWithENSName"
+                                       "wakuext_buildContact")
```

Comment:
just wondering why not the same method but with additional parameter?

Reply:
I think same method and additional parameter (ideally we use a struct), is a bit neater, I agree, but either way is ok I guess

Reply:
fixed